lib/tracker_api/resources/story.rb

```diff
@@ -121,6 +122,18 @@ def owners(params = {})
         end
       end
 
+      # Provides a list of all the transitions of the story.
+      #
+      # @param [Hash] params
+      # @return [Array[StoryTransition]]
+      def story_transitions(params = {})
```

Comment:
Cool. So the test .json is now generated. But the story that's used doesn't have any transitions:

request:

```
"request":{  
            "method":"get",
        "uri":"https://www.pivotaltracker.com/services/v5/projects/1027488/stories/66728004/transitions",
```

response:

[  "response":{  
            "status":{  
               "code":200,
               "message":null
            },](url)